app/models/street_art.rb

```diff
@@ -1,13 +1,7 @@
 class StreetArt < ApplicationRecord
   before_save :validate_location
   belongs_to :user
-  serialize :image_urls, Array
-  validates_presence_of :latitude
-  validates_presence_of :longitude
-  validates_presence_of :address
-  validates_presence_of :city
-  validates_presence_of :state
-  validates_presence_of :zipcode
+  # serialize :image_urls, Array
```

Comment:
Still TBD what form these will come back from the FE, so saving this if we need to handle an array...though we may need to push the burden into the GraphQL mutation types